packages/__tests__/Storyshots.test.ts

```diff
@@ -0,0 +1,12 @@
+import initStoryshots from '@storybook/addon-storyshots';
```

Comment:
Hvor skal vi legge mappene __tests__ og __mocks__? Skal vi endre strukture til src/packages/@navikit, og så ha tests og mocks under src/ ? @erlendev

Reply:
Kan bilde-snapshotene ligge så nære koden som testes som mulig, uten at det støyer det til for koden vi har i hhv `src` og `dist`? Tenker noe ala dette:

```
packages/@navikt/header/src
packages/@navikt/header/dist
packages/@navikt/header/__image_snapshots__/*.png
```

Eller må de ligge sammen med `__tests__` og `__mocks__`?

Reply:
☝️ For meg ser det ut som at ting som ligger i `__tests__` og `__mocks__` nå ikke har noe å gjøre med testing av enkeltmoduler. Men skal de kunne inneholde det, eller hvordan fungerer det? Hvis ikke tenker jo jeg at `__tests__` og `__mocks__` bør ligge på rot.

Reply:
tests og mocks inneholder nå bare filer som gjelder for alle stories, som konfigurasjon

Reply:
Jeg flytta tests og mocks til rot